cmd/cloudflared/tunnel/cmd_test.go

```diff
@@ -15,3 +18,76 @@ func TestHostnameFromURI(t *testing.T) {
 	assert.Equal(t, "", hostnameFromURI("trash"))
 	assert.Equal(t, "", hostnameFromURI("https://awesomesauce.com"))
 }
+
+func TestShouldRunQuickTunnel(t *testing.T) {
```

Comment:
I've scoped this specifically to quick tunnels, so it won't test falling through to other tunnel types.